tests/operon.test.ts

```diff
@@ -202,8 +202,19 @@ describe("operon-tests", () => {
 });
 
 class OperonTestClass {
+
+  static initialized = false;
   static cnt: number = 0;
 
+  // eslint-disable-next-line @typescript-eslint/require-await
+  @OperonInitializer()
+  static async init(_ctx: InitContext) { 
```

Comment:
underscore prefixes unused arguments. In this case we use the context so we should not prefix the variable name with an underscore.